src/org/traccar/reports/ReportUtils.java

```diff
@@ -0,0 +1,22 @@
+package org.traccar.reports;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import org.traccar.Context;
+
+public final class ReportUtils {
+
+    private ReportUtils() {
+    }
+
+    public static Collection<Long> getReportedDevices(Collection<Long> deviceIds, Collection<Long> groupIds) {
```

Comment:
I don't like the name of the function. It's a bit confusing. Can we rename it to something like `getDeviceList` or maybe `getSelectedDevices`.